prac_06/programming_language.py

```diff
@@ -0,0 +1,25 @@
+"""CP1404 Practical - programming language class
+Estimated time: 30 minutes
+Actual time: 35 minutes """
+
+
+class ProgrammingLanguage:
+    def __init__(self, name="", typing="", reflection="", year=0):
+        """
+        Initialise a Programming language instance.
+        name: string, the name of the programming language
+        typing: string, is the language dynamic or static
+        reflection: string, the language is reflective
```

Comment:
I concluded that self.reflection is a Boolean (True/False), not a string?

Reply:
Thanks, fixed.